protoc-gen-twirp_python/main.go

```diff
@@ -130,7 +131,7 @@ func (g *generator) generateProtobufClient(file *descriptor.FileDescriptorProto,
 	}
 	g.P(`    DEFAULT_HEADERS = {"Content-Type": "application/protobuf"}`)
 	g.P()
-	g.P(`    def __init__(self, server_address, headers_provider=None, request_hook=None, response_hook=None):`)
+	g.P(`    def __init__(self, server_address, headers_provider=None, request_hook=None, response_hook=None, verify_ssl=True):`)
```

Comment:
nit: please rename to `verify`, to be consistent with `requests` lib. DAI also follows that naming convention.

Reply:
Fixed